cx-portal-shared-components/README.md

```diff
@@ -1,3 +1,10 @@
+# Deprecated!
+
+Note that this version of the library is discontinued and development will continue only in
```

Comment:
I find the term 'version' a bit misleading, what to do you think of:
'Please note that this library is discontinued and development will continue only in:'

Reply:
done